api/src/test/java/gov/cms/ab2d/api/controller/BulkDataAccessAPIIntegrationTests.java

```diff
@@ -131,6 +131,30 @@ public void testBasicPatientExport() throws Exception {
         Assert.assertEquals(job.getUser(), userRepository.findByUsername(TEST_USER));
     }
 
+    @Test
+    public void testBasicPatientExportWithHttps() throws Exception {
+        ResultActions resultActions = this.mockMvc.perform(
+                get(API_PREFIX + FHIR_PREFIX + PATIENT_EXPORT_PATH).contentType(MediaType.APPLICATION_JSON)
+                        .header("Authorization", "Bearer " + token)
+                        .header("X-Forwarded-Proto", "https"))
+                .andDo(print());
```

Comment:
Let's just make sure this does not print too much to console

Reply:
Fair enough, but it's been there forever and I actually found it really helpful.